src/ui/controls/color-picker/index.tsx

```diff
@@ -40,8 +40,15 @@ function ColorPicker(props: ColorPickerProps) {
         const value = rawValue.trim();
         if (isValidColor(value)) {
             props.onChange(value);
+            props.onChange('');
```

Comment:
It's used to clear/reset the cssVariable when a valid color has been entered. Otherwise, every time the user reopens Firefox the colors would be instead be scraped from the CSS and change the theme they set.